src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/RateLimitPartition.T.cs

```diff
@@ -25,6 +25,9 @@ public RateLimitPartition(TKey partitionKey, Func<TKey, RateLimiter> factory)
         /// </summary>
         public TKey PartitionKey { get; }
 
-        internal readonly Func<TKey, RateLimiter> Factory;
+        /// <summary>
+        /// The function called when a rate limiter for the given <paramref name="partitionKey"/> is needed.
+        /// </summary>
+        public Func<TKey, RateLimiter> Factory { get; }
```

Comment:
Can you add at least minimal tests that validate the factory supplied to the ctor is roundtripped through this property?

Reply:
https://github.com/dotnet/runtime/pull/71145/commits/6ef0e911900900fc21afb15cd9a9e74f76d9fcec